### DISCUSSION TITLE
Maybe there is a way to merge the effort with `styx`

### DISCUSSION THREAD

**Discussion by blaggacao** [danth/coricamu#18] [General]
I've been meaning to reform `styx` for some time now, without really finding the time. A new beginning also seems promising, though.

To avoid the risk of going maintenance stale we should try to make this a community effort from the start (good example `cachix/devenv`)

Of particular interest might be: https://github.com/styx-static/styx/tree/master/src/renderers

**Comment by danth** [comment]
Hi, sorry I took a while to reply to this.

I agree, it would be great to get more community contributions here!

One of my reasons for creating Coricamu was that I found Styx a little confusing to get started with. I tried to make Coricamu simpler by using modules, so most features stay out of the way and use a reasonable default.

However Styx obviously has more features, some of which could be useful here too.

**Reply by blaggacao** [reply]
Since `styx` predates the module system, if it was rewritten to use it, instead, would that have an impact?

**Reply by danth** [reply]
It's certainly something worth thinking about - I'm not sure what's easier though, between rewriting Styx or starting from scratch here.

**Reply by blaggacao** [reply]
Oh, without any doubt, starting from scratch is _always_ easier :smile:.

I suspect there is also a long accumulated goodwill / brand / awareness with `styx`. Recreating that, on the other hand, might not be as easy.
